scripts/release/release-test.R

```diff
@@ -513,22 +540,30 @@ if(service_location == "") {
   }
 }
 
-cat("Do you have testfile [gse66351_1.rda] downloaded? (y/n) ")
-rda_available = readLines("stdin", n=1)
-rda_dir = ""
-test_resource = "gse66351_1.rda"
-if (rda_available == "y"){
-  cat(sprintf("Specify path to %s (including filename): ", test_resource))
-  rda_dir = readLines("stdin", n=1)
-} else {
-  cat("Where do you want to download it: ")
-  download_dir = readLines("stdin", n=1)
-  download_dir <- add_slash_if_not_added(download_dir)
-  if(dir.exists(download_dir)){
-    cli_alert_info(sprintf("Downloading %s into %s", test_resource, download_dir))
-    download.file("https://github.com/isglobal-brge/brge_data_large/raw/master/data/gse66351_1.rda", paste0(download_dir, test_resource))
-    rda_dir = paste0(download_dir, test_resource)
+rda_dir = user = Sys.getenv("RDA_TEST_DIR")
+if(rda_dir == ""){
+  # FIXME: Similar to above
+  # - we can define a directory for downloading files
+  # - if file 'gse66351_1.rda' does not exists there download it
```

Comment:
As all the `.parquet` files should already be available in `<root>/data/shared-lifecycle`, only the RDA file should be downloaded somewhere, which should be defined within the dev.env.dist config file. Maybe check if within the config file the filename is already present, and if not try to check if the path given + filename exists. If not, download.